src/reason-parser/reason_pprint_ast.ml

```diff
@@ -2859,12 +2859,6 @@ class printer  ()= object(self:'self)
   method type_variant_leaf1 opt_ampersand polymorphic print_bar x =
     let {pcd_name; pcd_args; pcd_res; pcd_loc; pcd_attributes} = x in
     let {stdAttrs} = partitionAttributes pcd_attributes in
-    let prefix = if polymorphic then "`" else "" in
-    let sourceMappedName = SourceMap (pcd_name.loc, atom (prefix ^ pcd_name.txt)) in
-    let nameOf = makeList ~postSpace:true [sourceMappedName] in
-    let barName =
```

Comment:
This logic is a duplicate from the `makeList` in `add_bar` below. Unified both into a new `add_bar`. Named the old `add_bar` into `add_bar_2`